tests/integration/plugins/index.js

```diff
@@ -17,6 +17,11 @@
  */
 // eslint-disable-next-line no-unused-vars
 module.exports = (on, config) => {
+  require("@cypress/code-coverage/task.js")(on, config);
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+
+  // It's IMPORTANT to return the config object
+  // with any changed environment variables
+  return config;
```

Comment:
I did not understand why the need for this came up now, like, how was it that everything was working before without returning the config object?